dask/array/core.py

```diff
@@ -1307,6 +1307,10 @@ def ravel(self):
 
     flatten = ravel
 
+    @wraps(np.choose)
+    def choose(self, choices):
+        return elemwise(variadic_choose, self, *choices)
```

Comment:
Would it make sense for this to just call the function implementation or vice versa?